WordPress/src/main/java/org/wordpress/android/ui/main/WPMainNavigationView.kt

```diff
@@ -137,35 +147,39 @@ class WPMainNavigationView @JvmOverloads constructor(
     override fun onNavigationItemReselected(item: MenuItem) {
         // scroll the active fragment's contents to the top when user re-taps the current item
         val position = getPositionForItemId(item.itemId)
-        if (position != PAGE_NEW_POST) {
+        if (position != getPosition(NEW_POST)) {
             (navAdapter.getFragment(position) as? OnScrollToTopListener)?.onScrollToTop()
         }
     }
 
     private fun getPositionForItemId(@IdRes itemId: Int): Int {
         return when (itemId) {
-            R.id.nav_sites -> PAGE_MY_SITE
-            R.id.nav_reader -> PAGE_READER
-            R.id.nav_write -> PAGE_NEW_POST
-            R.id.nav_me -> PAGE_ME
-            else -> PAGE_NOTIFS
+            R.id.nav_sites -> getPosition(MY_SITE)
+            R.id.nav_reader -> getPosition(READER)
+            R.id.nav_write -> getPosition(NEW_POST)
+            R.id.nav_me -> getPosition(ME)
+            else -> getPosition(NOTIFS)
         }
     }
 
+    private fun getPageForItemId(@IdRes itemId: Int): PageType {
+        return getPageType(getPositionForItemId(itemId))
```

Comment:
I think I'd move the logic from `getPositionForItemId` to this method and call `getPosition(getPageForItemId(itemId))` there instead. What do you think? It feels a bit cleaner (and you don't have to call `getPosition` there multiple times.

Reply:
Definitely cleaner; changed here 5d14346b530edc53485582cc0225262fcb362428